pymc/tests/test_distributions_random.py

```diff
@@ -428,9 +428,9 @@ class TestFlat(BaseTestDistributionRandom):
     ]
 
     def check_rv_inferred_size(self):
-        sizes_to_check = self.sizes_to_check or [None, (), 1, (1,), 5, (4, 5), (2, 4, 2)]
```

Comment:
But this will override `self.sizes_to_check` which the previous code didn't do, is that safe?

Reply:
That's of point of the or statement isnt it?

Reply:
Ok I see what youre saying. If its the assignment on self that`s concerning I'll change the PR to purely work off of local variables. All I care about is removing the `or` which doesn't add anything, just makes it harder to trace this test and understand what its doing.

In that case 
```
        sizes_to_check = [None, (), 1, (1,), 5, (4, 5), (2, 4, 2)]
        sizes_expected = [(), (), (1,), (1,), (5,), (4, 5), (2, 4, 2)]
        for size, expected in zip(sizes_to_check, sizes_expected):
            pymc_rv = self.pymc_dist.dist(**self.pymc_dist_params, size=size)
            expected_symbolic = tuple(pymc_rv.shape.eval())
            assert expected_symbolic == expected
```

Reply:
This is changing the original behavior, and ignoring when a test uses custom sizes_to_check

Reply:
yeah that looks good